map/set-max-extent/README.metadata.json

```
@@ -0,0 +1,29 @@
+{
+    "category": "Map",
+    "description": "Limit the view of a map to a particular area.",
+    "ignore": false,
+    "images": [
+        "SetMaxExtent.png"
+    ],
+    "keywords": [
+        "extent",
+        "limit panning",
+        "map",
+        "mapview",
+        "max extent",
+        "zoom",
+        "ArcGISMap",
+        "Envelope"
+    ],
+    "redirect_from": [
+        "/java/latest/sample-code/set-max-extent.htm"
+    ],
```

the `redirect_from` key in this instance can be left clear, since this is a new sample. This key is for samples for which a url may have existed already: e.g. we are going through a samples process of merging similar samples in to one. We would therefore copy those original sample URLS into redirect_from, so that if a user went to that original link, it wouldn't result in a 404 and would instead take them to the new sample. 

In this instance, the correct notation for there being no redirect is: 

```suggestion
    "redirect_from": "",
```